Proposed/Adding_An_Index/Step_2/MFIV/README.md

```diff
@@ -0,0 +1,184 @@
+# Requires 'jq' to be installed -- "brew install jq" (https://stedolan.github.io/jq/download/)
+#
+fetch_fleek() {
+    local timestamp="${1}T${2}Z"
+    set -o pipefail # Set pipefail option so if curl fails the script will stop
+    if ! curl --silent -L "https://storageapi.fleek.co/volatilitycom-bucket/indices/methodology=mfiv/interval=14d/currency=ETH/at=$timestamp/evidence.json" | jq .; then
+        echo "Command exited with code $?"
+    fi
+    set +o pipefail
+}
+```
+
+Here's an example of fetching data and writing it to a file:
+
+```
+$ fetch_fleek 2022-01-27 01:00:00 2>&1 | tee evidence.json 
+```
+
+**Programatically Create URL**
+
+The following script will make a header request to `fleek` to get the IPFS hash returned in the `etag` header value and won't actually download anything. Instead, it constructs the URL for IPFS.
+
+```
+#
+# Requires 'jq' to be installed -- "brew install jq" (https://stedolan.github.io/jq/download/)
+#
+fetch_ipfs() {
+    local timestamp="${1}T${2}Z"
+    set -o pipefail # Set pipefail option so if curl fails the script will stop
+    if ! curl -L -sI "https://storageapi.fleek.co/volatilitycom-bucket/indices/methodology=mfiv/interval=14d/currency=ETH/at=$timestamp/evidence.json" | tr -d '\r' | awk 'BEGIN {FS=": "}/^etag/{printf $2}' | tr -d '"' | awk '{printf "https://ipfs.io/ipfs/%s\n", $1}'; then
+        echo "Command exited with code $?"
+    fi
+    set +o pipefail
+}
+```
+
+Here's an example of fetching the URL:
+
+```
+$ fetch_ipfs 2022-01-27 01:00:00
+```
+
+Here's an example output URL. Notice it uses the IPFShash.
+
+```
+https://ipfs.io/ipfs/bafybeicwobtev44ivytnii24myc6fhxgpcimxnryneqlghbg4f2vcf6ugy
+```
+
+Download the data and name evidence.json and add into the mfiv_implementation folder.
+
+**Manually Create URL**
+
+If you have the `<IPFShash>` you can simply create the url with the following:
+
+`http://ipfs.fleek.co/ipfs/<IPFShash>`
+
+Or you can download the data by:
+
+`$ curl https://ipfs.io/ipfs/<IPFShash>`
+
+If you do not have the `<IPFShash>` you can create the URL with the following URL by changing `$timestamp`:
+
+```
+https://storageapi.fleek.co/volatilitycom-bucket/indices/methodology=mfiv/interval=14d/currency=ETH/at=$timestamp/evidence.json
+```
+
+Where
+
+- `$timestamp` - is written as the UTC ```DATE + "T" + TIME + "Z"```  to the second (MS are not included). 
+
+For example you would write 2022-01-27 01:00:00 UTC as:
+
+```2022-01-27T01:00:00Z```
+
+Which would give the following URL:
+
+```
+https://storageapi.fleek.co/volatilitycom-bucket//indices/methodology=mfiv/interval=14d/currency=ETH/at=2022-01-27T01:00:00Z/evidence.json
+```
+
+Download the data and add into the mfiv_implementation folder.
+
+## Verifying an Index
+
+The open-source code in this repo will ingest the ipfs.json input data and verify that the outputs are the same.
+
+1. Place the `ipfs.json` file in this directory. If you downloaded using a URL you can rename the hash or timestamp as ipfs.
```

Comment:
Rename ipfs.json as evidence.json

Reply:
All references to ipfs.json have been updated to evidence.json